src/main/java/godaddy/registry/jtoolkit2/session/TLSContext.java

```diff
@@ -39,6 +39,14 @@
     private final Logger maintLogger;
     private final Logger debugLogger;
 
+    protected TLSContext() {
+        String pname = getClass().getPackage().getName();
+        userLogger = Logger.getLogger(pname + ".user");
+        supportLogger = Logger.getLogger(pname + ".support");
+        maintLogger = Logger.getLogger(pname + ".maint");
+        debugLogger = Logger.getLogger(pname + ".debug");
```

Comment:
The new constructor is good. Could you please remove the duplicated lines (77 - 81 in the current file) from the existing constructor, and change it to call `this()` ?

Reply:
Done